controllers/btpoperator_controller_test.go

```diff
@@ -357,7 +363,7 @@ var _ = Describe("BTP Operator controller", Ordered, func() {
 			err := k8sClient.Delete(ctx, configMap)
 			Expect(err).To(BeNil())
 		})
-	})
+	})*/
```

Comment:
nit: I think we shouldn't have commented out tests in the repo for too long. If you intend to work on this as a follow-up task, that is fine, but otherwise, I would advocate for either adjusting the test to the new workflow or removing it

Reply:
I'm going to refactor the tests in the next PR with changes to update workflow, it's a follow-up task.